rootconf/u-boot-arch/sysroot-lib-onie/env-arch

```diff
@@ -0,0 +1,36 @@
+#  Copyright (C) 2018 Curt Brune <curt@cumulusnetworks.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+# Set ONIE environment variables.  The underlying store uses fw_setenv.
+#
+# arg $1 - variable name
+# arg $2 - variable value [optional]
+#
+# If the variable value is not specified the effect is to delete the
+# named variable.
+onie_setenv()
+{
+    local var="$1"
+    local val="$2"
+
+    if [ -n "$val" ] ; then
+        fw_setenv -f "$var" "$val"
+    else
+        # delete $var
+        fw_setenv -f "$var"
+    fi
+}
+
+# Get an ONIE environment variable.  The underlying store uses
+# fw_printenv.
+#
+# arg $1 - variable name
+#
+# If the variable name is not found the empty string "" is returned.
+onie_getenv()
```

Comment:
The u-boot arch `onie_setenv` and `onie_getenv` seems not to be applied to any place such as the changes in 10cfd9e3c64c837ab1ebe408e662a10069d360fb.